app/src/lib/stores/app-store.ts

```diff
@@ -3819,13 +3819,13 @@ export class AppStore extends TypedBaseStore<IAppState> {
       head.gitHubRepository.cloneURL === gitHubRepository.cloneURL
 
     if (isRefInThisRepo) {
-      const defaultRemote = await getDefaultRemote(repository)
+      const gitStore = this.getGitStore(repository)
+      const defaultRemote = gitStore.defaultRemote
```

Comment:
This is repeated above, may be worth breaking this out into a function...

Reply:
I'm going to push back on this change because creating more private functions inside `AppStore` that depend on `this.` is going to add more noise, particularly when it's only replacing two lines of code used in two places.